5.1.I_have_no_vinigret.py

```diff
@@ -0,0 +1,36 @@
+import random
+import datetime
+
+
+def rand_date(str_date1, str_date2):
+    """
+    The function get two strings of time in the format <%Y-%m-%d>
+    and returns a random date between them.
+    :param str_date1: first date
+    :param str_date2: second date
+    :return: random date between (str_date1, str_date2)
+    """
+    date1 = datetime.datetime.strptime(str_date1, '%Y-%m-%d')
+    date2 = datetime.datetime.strptime(str_date2, '%Y-%m-%d')
+    if date1.time() <= date2.time():
+        return date1 + (date2 - date1) * random.random()
+    else:
+        return None
+
+
+def main():
+    str_date1 = input('Enter first date:\n')
```

Comment:
Consider using different variable names such as start_date and end_date
https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0